doc/sphinx-guides/source/installation/config.rst

```diff
@@ -957,28 +987,7 @@ Logging & Slow Performance
      - When set to true, all JDBC calls will be logged allowing tracing of all JDBC interactions including SQL.
      - ``false``
 
-Database Configuration Tips
-+++++++++++++++++++++++++++
-
-In this section you can find some example scenarios of advanced configuration for the database connection that can improve service performance and availability.
-
-Database Connection Recovery
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Consider the following scenario: if there is no advanced configuration for the database connection and the Dataverse server loses that connection, for example if the database host is down, the server will be "dead" even after the database server is back to normal.
-The only solution to recover Dataverse would be to restart the service. To avoid this situation, the following settings can be used to configure validation of the database connection. 
```

Comment:
Why is all of this nice database connection recovery stuff being removed? It was added in this PR:

- #11730

Reply:
I redid this pull request. Resolving.